core/drivers/widevine_huk.c

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2023, The ChromiumOS Authors
+ */
+
+#include <kernel/tee_common_otp.h>
+#include <libfdt.h>
+#include <stdint.h>
+#include <string.h>
+
+static uint8_t dt_huk[HW_UNIQUE_KEY_LENGTH];
+static bool dt_huk_initialized;
+
+static TEE_Result init_widevine_huk_dt_data(void)
+{
+	int node = 0;
+	int len = 0;
+	void *fdt = NULL;
+	const void *value = NULL;
+
+	if (dt_huk_initialized)
+		return TEE_SUCCESS;
+
+	fdt = get_secure_dt();
+	if (!fdt)
+		return TEE_ERROR_NO_DATA;
+
+	node = fdt_path_offset(fdt, "/options/op-tee/widevine");
+	if (node < 0)
+		return TEE_ERROR_ITEM_NOT_FOUND;
+
+	value = fdt_getprop(fdt, node, "google,optee-hardware-unique-key", &len);
```

Comment:
From the DT bindings [posted to dt-schemas.org](https://lore.kernel.org/all/20231110043241.2920553-1-yich@chromium.org/), "google,optee-hardware-unique-key" is a required property. Does that means `CFG_WIDEVINE_HUK=y` is also required when `CFG_WIDEVINE_PTA` is enabled?

Reply:
Added CFG_WIDEVINE_HUK to the dependency of CFG_WIDEVINE_PTA.